src/nfts/types.ts

```diff
@@ -0,0 +1,26 @@
+interface NftMetaDataAttribute {
+  trait_type: string
+  value: string
+}
+
+interface NftMedia {
+  raw: string
+  gateway: string
+}
+
+export interface Nft {
+  tokenId: string
+  contractAddress: string
+  tokenUri?: string | null
+  ownerAddress?: string | null
+  metadata: {
+    name: string
+    description: string
+    image: string
+    dna?: string | null
+    id?: number | null
+    date?: number | null
```

Comment:
What's this date?
Can we rename it to something more meaningful, for instance `createdAt` or `indexedAt`, etc?

Reply:
It's an optional field on the Nft metadata I've seen populated as in the examples below. I'd be hesitant to rename it `createdAt` or `indexedAt` as it doesn't exist on all Nfts. We could add a property indexedAt form the indexer returned a level up in the Nft object.
```
{
    ...
    "metadata": {
        "name": "Celostrial #6294",
        "description": "The worlds first nfETs",
        "image": "ipfs://Qme97ifAexrMDfjE3DZRMasWhe7D276uYsCGonyprSA2MJ/6294.png",
        "dna": "77c251ffa01132e1cf747afd6f7da93b3a37a6ae",
        "id": 6294,
        "date": 1644389403533,
        "attributes": [...]
    },
    "media": [...]
},
        ```